src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -0,0 +1,416 @@
+import React from "react";
+import * as shortid from "shortid";
+import { IAssetMetadata, IRegion, RegionType,
+        AssetState, EditorMode, IProject } from "../../../../models/applicationState";
+import { CanvasTools } from "vott-ct";
+import { Editor } from "vott-ct/lib/js/CanvasTools/CanvasTools.Editor";
+import { RegionData, RegionDataType } from "vott-ct/lib/js/CanvasTools/Core/RegionData";
+import { TagsDescriptor } from "vott-ct/lib/js/CanvasTools/Core/TagsDescriptor";
+import { Point2D } from "vott-ct/lib/js/CanvasTools/Core/Point2D";
+import { Tag } from "vott-ct/lib/js/CanvasTools/Core/Tag";
+
+export interface ICanvasProps {
```

Comment:
Would recommend adding `selectedTags` here to handle integration between tags selected in the editor page vs. how the canvas reacts to that.